web/src/features/fba/components/map/fbaMap.test.tsx

```diff
@@ -3,10 +3,16 @@ import store from 'app/store'
 import FBAMap from 'features/fba/components/map/FBAMap'
 import { RunType } from 'features/fba/pages/FireBehaviourAdvisoryPage'
 import { DateTime } from 'luxon'
-import React from 'react'
+
 import { Provider } from 'react-redux'
+// @ts-ignore
+import { fetch } from 'whatwg-fetch'
 
 describe('FBAMap', () => {
+  beforeEach(() => {
```

Comment:
Why do we need to overwrite the global fetch function?

Reply:
in create react app version this was also being done, just under the hood. without this, when olpmtiles does a fetch to that s3 bucket it will expect browser fetch but will actually get node built in fetch and error out. this was a fascinating discovery that took me so many hours to figure out. other options would be to mock some sub components i suppose. or the whole fetch response.

Reply:
Interesting... would it make sense to replace the node fetch in an `after` function? Or perhaps just set this as the global fetch for all tests in `setupTests`?

Reply:
sure thing. done